content/millennium/r4/financial/charge-item.md

```diff
@@ -83,13 +83,18 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?  | Type          | Description
------------------------|------------|---------------|-------------------------------------------------------
- `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+ Name                  | Required?       | Type          | Description
+-----------------------|-----------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id     | [`token`]     | Encounter associated with event.
+[`_count`]             | No              | [`number`]    | The maximum number of results to return.
```

Comment:
nit: Let's put the paging parameter at the bottom of the list.

Reply:
+1

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/66cb49e49a70cc2e124fb6348991af2240402aa6#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR91